src/test/java/vertex/pro/edu/soung_box_app/repository/SongRepositoryTest.java

```diff
@@ -42,26 +58,58 @@ void findsSongsByGenre() {
         entityManager.flush();
         entityManager.clear();
 
-        List<SongEntity> entities = repository.findByParams(entity.getGenre());
+        List<SongEntity> entities = repository.findByParams(entity.getGenre(), null);
 
         assertThat(entities).isNotEmpty()
                 .extracting(SongEntity::getId)
-                .containsExactly(entity.getId());
+                .contains(entity.getId())
+                .doesNotContain(entity.getArtist());
+    }
+
+    @Test
```

Comment:
Your tests don't have a check for filtering.
Please save multiple entities with different artist & genre to DB and check that only entities with specified parameters are returned